File: help/pt_pt/exercicios_da_ficha_11/Ficha11_015/Ficha11_015.java
// Programador           -

import java.util.Scanner;

public class Ficha11_015 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Altura   : " );
        int altura = Integer.valueOf(keyb.nextLine());
        System.out.print("Largura  : " );
        int largura = Integer.valueOf(keyb.nextLine());
        System.out.print("Limite   : " );
        String limite = keyb.nextLine();
        System.out.print("Interior : " );
        String interior = keyb.nextLine();
        caixa(largura , altura , limite , interior ) ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha( int n , String c){
        for( int i = 0; i < n; i = i + 1 ) {
            System.out.print(c );
        }
        System.out.print("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void interior( int n , String cl , String ci){
        System.out.print(cl );
        for( int i = 1; i < n - 1; i = i + 1 ) {
            System.out.print(ci );
        }
        System.out.print(cl + "\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void caixa( int larg , int alt , String cl , String ci){
        linha(larg , cl ) ;
        for( int i = 1; i < alt - 1; i = i + 1 ) {
            interior(larg , cl , ci ) ;
        }
        linha(larg , cl ) ;
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:14

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_015/Ficha11_015.php
<?php
// Programador           -

//Main Function
function main() {
    echo  "Altura   : " ;
    $altura = intval(isset($_POST["altura"])? $_POST["altura"] : isset($_GET["altura"]) ? $_GET["altura"] : 0);
    
    echo $altura . "<br/>";// print input 
    echo  "Largura  : " ;
    $largura = intval(isset($_POST["largura"])? $_POST["largura"] : isset($_GET["largura"]) ? $_GET["largura"] : 0);
    
    echo $largura . "<br/>";// print input 
    echo  "Limite   : " ;
    $limite = isset($_POST["$limite"])? $_POST["$limite"] : isset($_GET["$limite"]) ? $_GET["$limite"] : 0;
    echo $limite . "<br/>";// print input 
    echo  "Interior : " ;
    $interior = isset($_POST["$interior"])? $_POST["$interior"] : isset($_GET["$interior"]) ? $_GET["$interior"] : 0;
    echo $interior . "<br/>";// print input 
    caixa($largura , $altura , $limite , $interior ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linha($n, $c) {
    for( $i = 0; $i < $n; $i = $i + 1 ) {
        echo $c  ;
    }
    echo  "\n" ;
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function interior($n, $cl, $ci) {
    echo $cl  ;
    for( $i = 1; $i < $n - 1; $i = $i + 1 ) {
        echo $ci  ;
    }
    echo $cl  .  "\n" ;
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function caixa($larg, $alt, $cl, $ci) {
    linha($larg , $cl ) ;
    for( $i = 1; $i < $alt - 1; $i = $i + 1 ) {
        interior($larg , $cl , $ci ) ;
    }
    linha($larg , $cl ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 12 20:14

?>